english/light-fire/xyzzy/stand-petrified/travel_to_Berlin/travel_to_Berlin.md

```diff
@@ -0,0 +1,8 @@
+You suddenly travel to Berlin 1989, lying on the road, at night (without wearing cloth!!).
```

Comment:
i continue with other people's one option.... so i leave it for other people adding option under my .md file